server/index.js

```diff
@@ -19,6 +19,19 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products', (req, res, next) => {
+  const sql = `
+    select *
+      from "products"
```

Comment:
This should not include the long description, I don't think.

Reply:
![Screen Shot 2020-06-12 at 11 18 05 AM](https://user-images.githubusercontent.com/60276373/84534422-b9309300-ac9e-11ea-8f3c-c88c75c7a2af.png)